docs/docs/react-without-jsx.md

```diff
@@ -33,16 +35,24 @@ class Hello extends React.Component {
 }
 
 ReactDOM.render(
-  React.createElement(Hello, {toWhat: 'World'}, null),
+  React.createElement(Hello, { toWhat: 'World' }, null),
   document.getElementById('root')
 );
 ```
 
+`React.createElement()` takes a component, a `props` object, and any amount of children. The given component can be in the form of a string (`React.createElement("div", null, null)`), a sub-class of `React.Component`, or a plain stateless-functional component.
```

Comment:
amount => number?

Reply:
The sentence about "the given component can be..." is a bit hard to read because of parens. Can we turn this into a bullet point list?